server/main.go

```diff
@@ -108,6 +110,18 @@ func main() {
 		glog.Fatalln("EVM Network ID is not set, call config.New() first")
 	}
 
+	tp, err := tracer.NewProvider(
+		tracer.WithEndpoint(cfg.Tracer.EndPoint),
+		tracer.WithInstanceID(cfg.Tracer.InstanceID),
+	)
```

Comment:
if tracer config is null, we should skip the integration of tracer

Reply:
https://github.com/iotexproject/iotex-core/blob/baeec450ad4cad3c8992213dbec55f97a0d94c50/pkg/tracer/tracer.go#L54-L65
Yes, it has been considered. if tracer config is null, tracer does not  sampling.

Reply:
If cfg.Tracer.EndPoint not configured, tracer.NewProvider should return (nil, nil) so that subsequent tracer operation could be simply skipped.
Since the tracer will be enabled on api server only, Don't introduce additional code logic on non-api server sounds better.

Reply:
+1, can the start/stop of tracer be moved into `StartServer`? or even better if it can be all handled inside api server's `Start/Stop`

Reply:
also, considering only API used tracer, is it possible to just add the `Endpoint string` to `API struct`?